### DISCUSSION TITLE
Noticed that audio bitrate is lower than desired

### DISCUSSION THREAD

**Discussion by TimothyChalamet** [aclap-dev/video-downloadhelper#528] [Help and Q&A]
### Your question

I'm trying to download vids where the audio bitrate is at 320aac, how do i enable this to download in this quality?

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): d153274991f8497b
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:126.0) Gecko/20100101 Firefox/126.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "arm64",
    "node": "18"
  },
  "home": "/Users/sammontgomery",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "n6.1-11-gc30f360955"
}

**Comment by GrampaWildWilly** [comment]
Let's have some URLs.  Also, I would recommend you get on the latest VDH 9 beta.  You can find it in the Wiki.  There's a link to that at the top of every page in this forum.
